### ISSUE TITLE
Windows Terminal is not showing in Context Menu in Windows 10

### ISSUE THREAD

**Issue by pradeep-repaka-mf** [microsoft/terminal#13651]
### Windows Terminal version

1.14.1962.0

### Windows build number

10.0.19044.0

### Steps to reproduce

I formatted my windows 10 OS yesterday and after reinstall OS installed the Windows Terminal using Microsoft Store, after that Windows Terminal is showing in Context Menu. Before formatting the OS Windows Terminal worked properly.

I tried to uninstall and install multiple times, but the behaviour is same.

### Expected Behavior

Windows Terminal should show in Context menu. 

### Actual Behavior

Windows Terminal not showing in Context menu.

**Comment by zadjii-msft** [comment]
* Have you restarted your PC since installing the Terminal/?
* What directory are you right-clicking on? IIRC there were bugs in the win10 handling of context menus for all of:
  * The desktop
  * right-clicking on the background of file explorer (not actually _on_ a directory)
  * on drives (e.g. directly on `C:\`)

**Comment by pradeep-repaka-mf** [comment]
> * Have you restarted your PC since installing the Terminal/?

Yes, I restarted. I installed and uninstalled multiple times and restarted multiple times also, but still in context menu did not appear.

> * What directory are you right-clicking on? IIRC there were bugs in the win10 handling of context menus for all of:
>   
>   * The desktop

I tried in multiple directories including desktop too.

>   * right-clicking on the background of file explorer (not actually _on_ a directory)

Yes. Once the required path of the directory opened in explorer, then right clicked on the empty area for the Terminal.

>   * on drives (e.g. directly on `C:\`)
I have only one Drive which is "C" and I tried in that also after you asked, still its same.

**Comment by zadjii-msft** [comment]
Hmm. This sounds like #6414, but I thought we had an out of band fix in #8638 for this. Hmm.

**Comment by pradeep-repaka-mf** [comment]
> Hmm. This sounds like #6414, but I thought we had an out of band fix in #8638 for this. Hmm.

Those are old versions and 2 years back. I am using latest version of terminal. Is there any to the Windows Terminal in context menu? Otherwise I have to manually type the entire directory path

**Comment by 89z** [comment]
I am having the same problem. I am on a work computer, so my profile looks like this:

~~~
PS C:\Users\spenny\Desktop> $PROFILE
\\TEMPOFS02\UserData$\spenny\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1
~~~

not sure if that makes a difference. I tried right clicking from the desktop, and shift right click. i installed using this:

https://github.com/microsoft/terminal/releases/download/v1.14.1962.0/Microsoft.WindowsTerminal_Win10_1.14.1962.0_8wekyb3d8bbwe.msixbundle

**Comment by zadjii-msft** [comment]
This might have been the same regression as #13523. Can you install https://github.com/microsoft/terminal/releases/tag/v1.15.2282.0, reboot, and see if that fixes this/?

**Comment by 89z** [comment]
trying the new version did not fix the problem for me. However I think my issue is #10119 - since it does work if you click on a folder, but not if you click a background

**Comment by zadjii-msft** [comment]
You know, I'm gonna combined dupe this to those two threads. If this wasn't fixed in [v1.15.228*](https://github.com/microsoft/terminal/releases/tag/v1.15.2282.0), then whatever it is, it's the same issue as /dup #10119. Thanks!
